README.md

```diff
@@ -0,0 +1,41 @@
+# <dataset> template
+
+Part of [`spinalcordtoolbox`](https://github.com/neuropoly/spinalcordtoolbox).
+
+
+## Usage
+
+1. Clone this repo.
+
+    ```
+    git mv src/spinalcordtoolbox/data/dataset src/spinalcordtoolbox/data/${dataset-name} 
+    cp ${data_files} src/spinalcordtoolbox/data/${dataset_name} 
+    vi setup.py # to edit name= and url=
+    vi README.md # to edit the title and *remove* this usage section
+    git add .
+    git commit
+    git remote add origin git@github.com:neuropoly/spinalcordtoolbox-data-${dataset_name}
+    git push
+    ```
+
+2. Go to https://github.com/neuropoly/spinalcordtoolbox-data-${dataset_name}/releases
+
+    1. Click "Draft Release"
+    2. Fill in a version tag. We use [this versioning policy](TODO)
+    3. Click "Publish Release"
+    4. Wait a few minutes;
+    5. Monitor the progress at https://github.com/neuropoly/spinalcordtoolbox-data-${dataset_name}/actions/workflows/release.yml
+    6. The release should appear on https://github.com/neuropoly/spinalcordtoolbox-data-${dataset_name}/releases
```

Comment:
Maybe this should go into its own "Usage" section. Right now, it gets removed when the `README.md` is edited, but it seems like it's worth keeping.

(e.g. split into `How to use this repo template` (which gets removed) and `How to use {dataset_name}` (which stays))